reference/5.1/Microsoft.PowerShell.Management/Start-Process.md

```diff
@@ -416,10 +425,14 @@ parameter. Otherwise, this cmdlet does not return any output.
 
 ## NOTES
 
-* This cmdlet is implemented by using the **Start** method of the **System.Diagnostics.Process**
+- This cmdlet is implemented by using the **Start** method of the **System.Diagnostics.Process**
   class. For more information about this method, see
   [Process.Start Method](/dotnet/api/system.diagnostics.process.start?view=netframework-4.7.2#overloads).
 
+- On Windows, when you use **UseNewEnvironment**, the new process starts only containing the default
```

Comment:
Should this note be here or anywhere other than 7.x since it's new functionality?

Reply:
No. It exists in all versions but the behavior changed in 7.1.